Firmware unable to time accurately

Curretnly the program thinks the MCU is running at 1 MHz, preventing timing functions from being accurate.

This is related to the compiler warning about `F_CPU`. There are 2 `F_CPU` warnings, the first is from a `# warning` directive:

`avr\include\util\delay.h:92:3: warning: #warning "F_CPU not defined for <util/delay.h>`

Reading this file gives additional details on what the library expects:

```
#ifndef F_CPU
/* prevent compiler error by supplying a default */
# warning "F_CPU not defined for <util/delay.h>"
/** \ingroup util_delay
    \def F_CPU
    \brief CPU frequency in Hz

    The macro F_CPU specifies the CPU frequency to be considered by
    the delay macros.  This macro is normally supplied by the
    environment (e.g. from within a project header, or the project's
    Makefile).  The value 1 MHz here is only provided as a "vanilla"
    fallback if no such user-provided definition could be found.

    In terms of the delay functions, the CPU frequency can be given as
    a floating-point constant (e.g. 3.6864E6 for 3.6864 MHz).
    However, the macros in <util/setbaud.h> require it to be an
    integer value.
 */
# define F_CPU 1000000UL
#endif
```

(The second warning is about the definition of `F_CPU` in `firmware.c` which appears to not be the correct way to do it, but both warnings should go away once this is addressed.)

--- a/firmware.c
+++ b/firmware.c
@@ -1,8 +1,6 @@
 #include <avr/io.h>
-#include <util/delay.h>
 #include "usb_serial.h"
 
-#define F_CPU 16000000UL  // 1 MHz
 #define LED_CONFIG (DDRD |= (1<<6))
 #define LED_ON (PORTD |= (1<<6))
 #define LED_OFF (PORTD &= ~(1<<6))

